mucommander-commons-util/src/main/java/com/mucommander/commons/util/ui/button/ButtonChoicePanel.java

```diff
@@ -74,9 +77,9 @@ public class ButtonChoicePanel extends JPanel implements KeyListener, FocusListe
      * @param nbCols   number of columns for the buttons grid, if <=0 all buttons will be put on a single row
      * @param rootPane associated with this ButtonChoicePanel
      */
-    public ButtonChoicePanel(JButton buttons[], int nbCols, JRootPane rootPane) {
-        this.buttons = buttons;
-        int nbButtons = buttons.length;
+    public ButtonChoicePanel(List<JButton> buttons, int nbCols, JRootPane rootPane) {
+        this.buttons = new ArrayList<>(buttons);
```

Comment:
what's the reason for cloning the list of buttons here?

Reply:
To ensure immutability of this list (if caller wanted to modify the list). I don't insist, especially that there's not so much risk as there is not so much of multi-threading around.

Reply:
yeah, that code as it is now suggests the list might changed elsewhere - if it's not the case (and it shouldn't be), I'd prefer to change it